packages/react-components/components/PhoneNumberInput.tsx

```diff
@@ -69,40 +69,56 @@ export default class PhoneNumberInput extends React.Component<Props, State> {
     }
   }
 
+  async getPhoneNumberFromNativePicker() {
+    // linter doesn't allow following code without splitting this code in two lines
+    const number = await SmsRetriever.requestPhoneNumber()
+    return number
+  }
+
   async triggerPhoneNumberRequestAndroid() {
     try {
-      const phone = await SmsRetriever.requestPhoneNumber()
-      const phoneNumber = parsePhoneNumber(phone, '')
+      let phone
+      try {
+        phone = await this.getPhoneNumberFromNativePicker()
```

Comment:
Any reason to use a function here rather than just directly using `SmsRetriever.requestPhoneNumber()`?

Reply:
To make mocking different results easier